taxonomic_dist_by_soil_layer/R/taxonomic_dist_soil_layer_R.ipynb

```diff
@@ -0,0 +1,2622 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.        collectio = 'biosample_set', 
</pre><p>It seems like this still runs, but isn't the argument named 'collection' ?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microbiomedata/nmdc_notebooks/pull/42/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
<p>I'm surprised it still runs!  Will fix.</p>